examples/chart/teleport-cluster/templates/deployment.yaml

```diff
@@ -211,7 +211,7 @@ spec:
         secret:
           secretName: "license"
 {{- end }}
-{{- if .Values.gcp.credentialSecretName }}
+{{- if and (.Values.gcp.credentialSecretName) (eq .Values.chartMode "gcp") }}
```

Comment:
Should the `(.Values.gcp.credentialSecretName)` condition be removed since the value is being required below?

Reply:
```suggestion
{{- if (eq .Values.chartMode "gcp") }}
```